install/kubernetes/cilium/values.yaml

```diff
@@ -3169,6 +3169,14 @@ authentication:
           labels: { }
           # -- SPIRE Workload Attestor kubelet verification.
           skipKubeletVerification: true
+          # -- SPIRE agent tolerations configuration
```

Comment:
You need to update `install/kubernetes/cilium/values.yaml.tmpl` and then run `make -C install/kubernetes`

Reply:
thank you, I was wondering how this would be done, manually seemed off to me :)